### PR TITLE
Setup unimported on live-common + ci

### PR DESCRIPTION
**Pull request by valpinkman**
- chore: add unimported to project for easier reuse
- chore: setup unimported on common

### 📝 Description

- install `unimported` on the root of the project so it can be used throughout the monorepo
- setup unimported on ledger live common (thanks @gre)
  - cleanup dead code
  - removed unused dependencies
- setup unimported on CI for libs codechecks

#### Tradeoffs

- ledger-live-common/deviceSDK is ignored for now, considered as possible entry point (instead of removing the unused code).
- in unimported config. we had to defined a bunch of config due to peerDependencies' deps

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: [LIVE-7944] 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-7944]: https://ledgerhq.atlassian.net/browse/LIVE-7944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ledger-live-common/src/hw/actions/uninstallAllApps.ts [hunk 1]
```diff
@@ -1,125 +0,0 @@
...
-        ...state,
-        unresponsive: false,
-        uninstallAppsRequested: false,
-        isLoading: false,
-        appsUninstalled: true,
-      };
-  }
-};
-
-export const createAction = (
```
Comment (alexandremgo) [inline libs/ledger-live-common/src/hw/actions/uninstallAllApps.ts:82]: Is it possible to keep this one ? It's associated command is `libs/ledger-live-common/src/hw/uninstallAllApps.ts`, which is only used in our cli. But Juan created 4 months ago just before leaving and we might use it

File: libs/ledger-live-common/src/hw/actions/initSell.ts [hunk 2]
```diff
@@ -1,188 +0,0 @@
-import { Observable, of, concat } from "rxjs";
-import { scan, tap, catchError } from "rxjs/operators";
-import { useEffect, useState } from "react";
-import type { ConnectAppEvent, Input as ConnectAppInput } from "../connectApp";
-import type { Action, Device } from "./types";
-import type { Transaction, TransactionStatus } from "../../generated/types";
-import type { AppState } from "./app";
-import { log } from "@ledgerhq/logs";
-import { createAction as createAppAction } from "./app";
-import type { InitSellResult, SellRequestEvent } from "../../exchange/sell/types";
```
Comment (alexandremgo) [inline libs/ledger-live-common/src/hw/actions/initSell.ts:10]: If this action is not used, i  think `InitSellResult` can be removed. But I don't have enough knowledge on those, it's up to PTX to decide

Comment (alexandremgo) [reply libs/ledger-live-common/src/hw/actions/initSell.ts:10]: Oh well `libs/ledger-live-common/src/exchange/sell/types.ts` is removed too ahah 🧹

**Review by alexandremgo — APPROVED** [review]
Thx 🧹🦖

File: libs/ledger-live-common/src/hw/actions/uninstallAllApps.ts (hunk 1 above)
Comment (valpinkman) [reply libs/ledger-live-common/src/hw/actions/uninstallAllApps.ts]: done

**Review by hzheng-ledger — APPROVED** [review]
looks good to me. Thanks

**Pull request merged**